Lw1/Converter/app/build.gradle

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
APK signing is not implemented. See requirements.

Reply:
Fixed